app/controllers/descriptives_controller.rb

```diff
@@ -42,7 +42,7 @@ def convert_metadata_fail(failure)
     render :new, status: :unprocessable_entity
   end
 
-  def load_and_authorize_cocina
+  def load_and_authorize_resource
```

Comment:
Is there a reason this isn't in ApplicationController? Then there is no need to worry about naming conventions.

Reply:
I have the same question.

Reply:
That's where I was going... but in small steps.